script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -317,6 +318,7 @@ Lighten up will ya, we're definitely going to kill this Dead Apostle.
 [sha:293f047a738a720399f1f66d2e1673fecfd67623]{
 -- Page 96, Offset 30814.
 -- ……で、どうなんだアカシャの蛇。<今|・><回|・><は|・><ち|・><ゃ|・><ん|・><と|・><思|・><い|・><出|・><し|・><た|・><か|・>？　<解|・><脱|・><の|・><法|・><は|・><ど|・><こ|・><に|・><あ|・><る|・>？”」
+// group pr check: ask petrikow
```

Comment:
@Petrikow

Reply:
might want to ask in discord. Dunno how often he checks github (and github emails tend to get lost in gmail's spam filter)